default/python/AI/buildings.py

```diff
@@ -0,0 +1,48 @@
+import freeOrionAIInterface as fo
+from enum import Enum
+from typing import List
+
+from common.fo_typing import PlanetId
+from freeorion_tools.caching import cache_for_current_turn, cache_for_session
+
+
+class Building(Enum):
+    """
+    Enum to represent fo building template.
+
+    Pros:
+    - Shorten code a bit
+    - Allow to have shorter names for ids,
+      we could get read of BLD prefix, since it always use with Building. prefix.
```

Comment:
Well, using the class name as a prefix, names rather get longer. But that is negligible.
Using python names instead of strings has lots of advantages. E.g. IDE can use name completion and show you were a building type is used, typos can be detected by flake.